roles/setup/templates/systemd_receptor.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Ansible Receptor for Linux
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+ExecStart={{ receptor_install_dir }}/receptor --config {{ receptor_config_path }}/receptor.conf
+StandardOutput=append:/var/log/receptor/receptor.log

```

Comment:
tried this out -- I believe this directory needs to exist, systemd won't create it. We can add a task to create this directory before starting the service

Reply:
we can even let users define this path if you think that is better